systers_portal/static/js/edit_profile.js

```diff
@@ -0,0 +1,19 @@
+var htmlInnerContent="<label for='id_profile_picture' \n" + 
+"class='control-label'>Profile picture</label><br>\n" +
+"<button id='upload_pic' class='btn btn-default'>\n" +
+"UPLOAD PICTURE</button><strong id='file_name_display'>\n" +
+ "</strong><div class='controls'> <input name='profile_picture'\n" +
+  "class='clearablefileinput' id='id_profile_picture' type='file'\n" +
+   "style='position:absolute; left:-999px;'></div>";
+
+$("#div_id_profile_picture").html(htmlInnerContent);
+
+$("#upload_pic").click(function() {
+	$("#id_profile_picture").click();
+});
+
+document.getElementById("id_profile_picture").onchange = function () {
+  var str = this.value;
+  var pathTokens = str.split("\\");
+  $("#file_name_display").html(" " + pathTokens[pathTokens.length - 1]);
+};
```

Comment:
You can add comments in this file about
1. What this file does and
2. What each component does
You can add a new line at the end of the file.